front-end/src/pages/CreateEvent.jsx

```diff
@@ -18,7 +18,7 @@ export default function CreateEvent() {
   // time zone imported from utilities 
   const [timeZone, setTimeZone] = useState('America/Adak');
   // event type = In-person or Virtual
-  const [eventType, setEventType] = useState('In-Person');
```

Comment:
if you change this old events won't work because they have been created with 'In-Person'. it's fine just know the search won't work until we change the data.